src/platform/graphics/render-pass.js

```diff
@@ -276,7 +276,7 @@ class RenderPass {
         }
 
         // color
-        const numColorOps = rt?._colorBuffers?.length ?? 1;
+        const numColorOps = rt ? rt._colorBuffers?.length : 1;
```

Comment:
Wait....is this right? If `rt` is defined but `rt._colorBuffers` is not, `numColorOps` will be set to `undefined` right?

Reply:
you're right, fixed, thanks